packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -190,6 +190,30 @@ Expected response body:
 }
 ```
 
+### Azure_Core_Basic_createUser
```

Comment:
what is the goal of this test? what does `createUser` mean as a scenario

Reply:
hello @timotheeguerin This test is to cover the senario: the request body and the response body is different from the resource type.

Reply:
So what specific things from the azure core library does this scenario test. Seems to me it’s basically just adding wrapper on top of what could just be a pure typespec operation